components/Navbar/ProfileDropdown.tsx

```diff
@@ -26,19 +29,19 @@ export default function ProfileDropDown() {
         aria-labelledby="hs-dropdown-with-header"
       >
         <div className="mt-2 py-2 first:pt-0 last:pb-0">
-          <a
+          <Link
             className="flex items-center gap-x-3.5 py-2 px-3 rounded-lg text-sm text-gray-800 hover:bg-gray-100 focus:ring-2 focus:ring-blue-500 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300"
-            href="#"
+            href="/profile"
           >
-            {`${firstName} ${lastName}`}
-          </a>
-          <a
+            Setting Page
+          </Link>
+          <Link
             className="flex items-center gap-x-3.5 py-2 px-3 rounded-lg text-sm text-gray-800 hover:bg-gray-100 focus:ring-2 focus:ring-blue-500 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300"
-            href="#"
+            href="/login"
           >
             <LuLogOut />
-            Logout
-          </a>
+            <button onClick={logout}>Logout</button>
+          </Link>
         </div>
```

Comment:
How is this possible? `Link` and `button` both work together after a single click! Think about race conditions which is a common problem in software engineering in many fields and technologies.